content/articles/fosdem-2020.md

```diff
@@ -0,0 +1,173 @@
+---
+title: "FOSDEM 2020"
+date: 2020-02-17
+type: "post"
+authors: ["nyby"]
+tags: ["linux", "conference", "events"]
+lede: "A summary of FOSDEM, the Free and Open source Software Developers' European Meeting."
+shortlede: ""
+poster: ""
+thumbnail: ""
+socmediaimg: ""
+hiliteimg: ""
+poster_sourceurl: ""
+poster_source: ""
+bookendanimal: ""
+---
+
+I attended [FOSDEM](https://fosdem.org/2020/) in Brussels on February 1st and 2nd, last
+weekend. It was pretty overwhelming, the amount of people throughout
+the [ULB](https://www.ulb.be/en) campus. The number of attendees is around 8,000, and at
```

Comment:
Sentence is a bit awkward. Maybe combine with the next one?

Reply:
I re-worded this to use less passive voice.